ethcore/src/miner/transaction_queue.rs

```diff
@@ -115,9 +115,7 @@ impl Ord for TransactionOrigin {
 		}
 
 		match (*self, *other) {
-			(TransactionOrigin::RetractedBlock, _) => Ordering::Less,
-			(_, TransactionOrigin::RetractedBlock) => Ordering::Greater,
```

Comment:
`(TransactionOrigin::Local, TransactionOrigin::RetractedBlock)` would give `Ordering::Greater` originally and `Ordering::Less` after this PR.